ui/src/layout/Dashboard/CounterDisplay/index.jsx

```diff
@@ -1,59 +1,82 @@
-import React, { useMemo } from "react";
-import { useFetch } from "hooks";
+import React from "react";
 import { formatNumber } from "utils/utils";
-
+import { useQuery } from "@tanstack/react-query";
 import COLORS from "../../../utils/scss_variables.module.scss";
-import { APIS } from "../../../utils/systemConsts";
-
+import { openClarityApi } from "../../../api/openClarityApi";
+import QUERY_KEYS from "../../../api/constants";
 import "./counter-display.scss";
 
-export const ScanCounterDisplay = () => {
-  const [{ data, error, loading }] = useFetch(APIS.SCANS, {
-    queryParams: {
-      $count: true,
-      $top: 1,
-      $select: "count",
-      $filter:
+export function ScanCounterDisplay() {
+  const { data, isError, isLoading } = useQuery({
+    queryKey: [QUERY_KEYS.scans],
+    queryFn: () =>
+      openClarityApi.getScans(
         "status/state eq 'Aborted' or status/state eq 'Failed' or status/state eq 'Done'",
-    },
+        "count",
+        true,
+        1,
+      ),
   });
 
   return (
     <div
       className="dashboard-counter"
       style={{ background: COLORS["color-gradient-green"] }}
     >
-      {loading || error ? (
-        ""
-      ) : (
+      {!isLoading && !isError && (
         <div className="dashboard-counter-content">
           <div className="dashboard-counter-count">
-            {formatNumber(data.count)}
+            {formatNumber(data.data.count)}
```

Comment:
you could use selectors to avoid the double data props
e.g.
```ts
const {data} = useQuery({
  // ...
  select: response => response.data,
});

// ...
{formatNumber(data.count)}
```